File: app/src/main/java/seveida/firetvforreddit/response/objects/Preview.java
package seveida.firetvforreddit.response.objects;

import java.util.List;
import com.squareup.moshi.Json;

public class Preview {

    @Json(name = "images")
    public List<Image> images = null;
    @Json(name = "enabled")
    public boolean enabled;

    /**
     * No args constructor for use in serialization
     *
     */
    public Preview() {
    }

    /**
     *
     * @param enabled
     * @param images
     */
    public Preview(List<Image> images, boolean enabled) {
        super();
        this.images = images;
        this.enabled = enabled;
    }

}

File: app/src/main/java/seveida/firetvforreddit/response/objects/Preview.kt
package seveida.firetvforreddit.response.objects

import com.squareup.moshi.Json


class Preview {

    @Json(name = "images")
    var images: List<Image>? = null
    @Json(name = "enabled")
    var enabled: Boolean = false

    /**
     * No args constructor for use in serialization
     *
     */
    constructor() {}

    /**
     *
     * @param enabled
     * @param images
     */
    constructor(images: List<Image>, enabled: Boolean) : super() {
        this.images = images
        this.enabled = enabled
    }

}
